test/client-errors.js

```diff
@@ -27,7 +27,7 @@ test('GET errors and reconnect with pipelining 1', (t) => {
     const client = new Client(`http://localhost:${server.address().port}`, {
       pipelining: 1
     })
-    t.tearDown(client.close.bind(client))
+    t.tearDown(client.destroy.bind(client))
```

Comment:
This tests don't actually finish the requests which means if we do close we have to wait for a timeout... just do destroy instead